src/components/UI/Timer/Timer.tsx

```diff
@@ -71,6 +71,7 @@ export const Timer = (props: TimerProps) => {
     const lastMessageTime = moment(time);
     const duration = moment.duration(currentTime.diff(lastMessageTime));
     hours = Math.floor(duration.asHours());
+    if (hours < 0) hours = 0;
```

Comment:
Sometimes the difference is coming up as -1. This results in hours becoming 25. Adding a check for that.